blueocean-core-js/src/js/i18n/i18n.js

```diff
@@ -130,3 +134,16 @@ export default function i18nTranslator(pluginName, namespace = toDefaultNamespac
 
     return translator;
 }
+
+/**
+ * Create i18n resources bound to a plugin name.
+ * @param pluginName
+ * @param i18nKeys
+ */
+export function mockTranslator(pluginName, i18nKeys = {}) {
+    const cacheKey = buildCacheKey(pluginName);
+    translatorCache[cacheKey] = function mockTranslate(key, opts = {}) {
+        const resolved = i18nKeys[key];
```

Comment:
Maybe for tests would it be even better to just return the translation key, maybe prefixed with something to indicate it was properly run through the translate function? Like... `return 'T/' + key`